onnxruntime/core/providers/shared/node_unit/node_unit.cc

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/graph/graph.h"
+// #include "core/graph/graph_viewer.h"
+
+#include "node_unit.h"
+
+namespace onnxruntime {
+
+NodeUnit::NodeUnit(const Node& node)
+    : nodes_{&node},
+      node_(node),
+      type_(Type::Node) {
+  InitForNode();
+}
+
+NodeUnit::~NodeUnit() = default;
```

Comment:
why do we need to explicitly specify the defaulted dtor?

Reply:
This was from the old code when it was an interface, can be deleted now